### ISSUE TITLE
unable to compile project with current Flutter beta 1.17.0-3.2.pre

### ISSUE THREAD

**Issue by ethael** [memspace/zefyr#281]
## Steps to Reproduce

1. upgrade flutter
2. compile project

## Logs

```
Compiler message:
../../../flutter/.pub-cache/hosted/pub.dartlang.org/zefyr-0.10.0/lib/src/widgets/input.dart:11:7: Error: The non-abstract class 'InputConnectionController' is missing implementations for these members:
 - TextInputClient.currentTextEditingValue
Try to either
 - provide an implementation,
 - inherit an implementation from a superclass or mixin,
 - mark the class as abstract, or
 - provide a 'noSuchMethod' implementation.

class InputConnectionController implements TextInputClient {
      ^^^^^^^^^^^^^^^^^^^^^^^^^
../../../flutter/packages/flutter/lib/src/services/text_input.dart:757:24: Context: 'TextInputClient.currentTextEditingValue' is defined here.
  TextEditingValue get currentTextEditingValue;
                       ^^^^^^^^^^^^^^^^^^^^^^^
Failed to compile application.
```

```
❯ flutter doctor -v
[✓] Flutter (Channel beta, v1.17.0-3.2.pre, on Linux, locale en_US.utf8)
    • Flutter version 1.17.0-3.2.pre at /opt/flutter
    • Framework revision 2a7bc389f2 (6 days ago), 2020-04-21 20:34:20 -0700
    • Engine revision 4c8c31f591
    • Dart version 2.8.0 (build 2.8.0-dev.20.10)

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /opt/android-sdk/
    • Platform android-29, build-tools 29.0.3
    • ANDROID_HOME = /opt/android-sdk/
    • Java binary at: /opt/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ✗ Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Android Studio (version 3.6)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 44.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[!] Android Studio
    • Android Studio at /opt/android-studio/
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • android-studio-dir = /opt/android-studio/
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] IntelliJ IDEA Ultimate Edition (version 2019.3)
    • IntelliJ at /opt/idea
    • Flutter plugin version 45.0.2
    • Dart plugin version 193.6911.31

[✓] Connected device (3 available)
    • Nokia 6 1   • android-arm64  • Android 10 (API 29)
    • Chrome     • chrome           • web-javascript • Google Chrome 81.0.4044.122
    • Web Server • web-server       • web-javascript • Flutter Tools

! Doctor found issues in 2 categories.
```

**Comment by kaumudpa** [comment]
+1

**Comment by MartinHlavna** [comment]
I am also experiencing this problem.

**Comment by gwxtr** [comment]
Same here, please fix.

**Comment by Arne1303** [comment]
I think this is a duplicate of #275 you can fix it for now by adding the following into zefyr-0.10.0/src/widgets/input.dart 

@override TextEditingValue get currentTextEditingValue => _lastKnownRemoteTextEditingValue;

It's just a temporary fix but it works... still annoying

**Comment by yzxben** [comment]
This now happens with the flutter stable channel, as of yesterday.

**Comment by essuraj** [comment]
Now also in stable https://pub.dev/packages/zefyr#-analysis-tab-

**Comment by Dangtrungphu** [comment]
+1

**Comment by pangboww** [comment]
+1

**Comment by ferysyukur** [comment]
+1

**Comment by ethael** [comment]
fixed with 0.11.0
